draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm

```diff
@@ -46,6 +47,17 @@ decoder keyCode =
         Other
 
 
+keyDownDecoder : Decoder Key
+keyDownDecoder =
```

Comment:
We can use Html.Events.keyCode instead!

Reply:
sweet, didn't know that was there!

Reply:
There is even a mention about the deprecation of keyCode which is pretty neat.

Reply:
It should mention this in the docs for `Browser.Events.onKeyDown` though! (it tells you to write your own decoder)